### DISCUSSION TITLE
Problem creating FILE Attribute

### DISCUSSION THREAD

**Discussion by ghost** [saleor/saleor#14344] [General]
Hi,

I am trying to create an attribute with inputType `FILE`. My use case is to have a "Brand" attribute with values `name` of the brand and `fileUrl` pointing to the logo of the brand. 

Based on the documentation, `FILE` attribute can be used to store a product manual as a PDF file or store a hero image for a page, which seems similar to my use case.

First, I tried the below mutation:
```
mutation{
  attributeCreate(input:{
    inputType: FILE,
    entityType: PRODUCT,
    name: "Brand",
    type: PRODUCT_TYPE,
    values: [
      {name: "brand1", fileUrl: "https://url/of/brand1"},
      {name: "brand2", fileUrl: "https://url/of/brand2"},
    ]
  }){
    attribute{
      id
    }
    errors{
      field
      message
    }
  }
}
```

However, I am getting the error "Values cannot be used with input type file."

I also tried amending the above mutation such that values is `[]`, and then do:
```
mutation{
  attributeValueCreate(
    attribute:"<brandAttributeId>", 
    input:{
      name:"brand1",
      fileUrl:"https://url/of/brand1"
    }){
    errors{
      message
      field
    }
    attribute{
      id
    }
  }
}
```

Now, I am getting the error "The field file_url can be defined only for swatch attributes."

What is the right way to create a "Brand" attribute with a list of values that contains the brand name and logo url?

Thanks in advance!

**Comment by timuric** [comment]
@wshinigamic are you able to recreate this issue using dashboard?
